server/src/test/kotlin/core/gameflow/PlayerTest.kt

```diff
@@ -147,4 +152,31 @@ class PlayerTest {
         )
         assert(betPlayer.isDecisive)
     }
+
+    @Test
+    fun `hand member should create a proper poker hand out of 5 best cards`() {
+        val player = Player(
+                position = 0,
+                stack = 0,
+                holeCards = listOf(Card(CardRank.ACE, CardSuit.SPADES), Card(CardRank.SIX, CardSuit.HEARTS))
+        )
+
+        val communityCards = listOf(
+                Card(CardRank.JACK, CardSuit.HEARTS),
+                Card(CardRank.TWO, CardSuit.CLUBS),
+                Card(CardRank.TWO, CardSuit.DIAMONDS),
+                Card(CardRank.SIX, CardSuit.CLUBS),
+                Card(CardRank.KING, CardSuit.DIAMONDS)
+        )
+
+        val properHand = TwoPair(listOf(
+                Card(CardRank.ACE, CardSuit.SPADES),
+                Card(CardRank.TWO, CardSuit.CLUBS),
+                Card(CardRank.TWO, CardSuit.DIAMONDS),
+                Card(CardRank.SIX, CardSuit.CLUBS),
+                Card(CardRank.SIX, CardSuit.HEARTS)
+        ))
+
+        assert(player.hand(communityCards).compareTo(properHand) == 0)
```

Comment:
Not writing tests is nice, but you know what is nicer? A working application.

Reply:
:(